config/settings.rb

```diff
@@ -15,7 +15,7 @@ class Settings < Hanami::Application::Settings
     setting :session_secret, constructor: Types::String
 
     # Assets
-    setting :precompiled_assets, constructor: Types::Params::Bool.optional.default(false)
-    setting :assets_server_url, constructor: Types::String.optional.default("http://localhost:8080")
+    setting :precompiled_assets, false, constructor: Types::Params::Bool.optional
```

Comment:
```suggestion
    setting :precompiled_assets, constructor: Types::Params::Bool.optional, default: false
```

(for the reason above)